velox/exec/Unnest.cpp

```diff
@@ -268,10 +281,17 @@ VectorPtr Unnest::generateOrdinalityVector(const RowRange& range) {
 
   VELOX_DCHECK_GT(range.size, 0);
 
+  vector_size_t index = 0;
   range.forEachRow(
-      [&](vector_size_t /*row*/, vector_size_t start, vector_size_t size) {
+      [&](vector_size_t row, vector_size_t start, vector_size_t size) {
+        if (isOuter_) {
```

Comment:
if (isOuter_ && awOrdinalityIsNull_[row])
And if we set the ordinalityVector in the index, do we need to set rawOrdinality? I think not